src/lib/structures/Josh.ts

```diff
@@ -79,6 +78,9 @@ export class Josh<StoredValue = unknown> {
       );
 
     this.middlewares = new MiddlewareStore({ instance: this });
+
+    if (middlewares !== undefined)
+      for (const middleware of middlewares.filter((middleware) => middleware instanceof Middleware)) this.use(middleware);
```

Comment:
Perhaps throw an error if an inputted "middleware" is not an instance of Middleware, or at least a warning